### PR TITLE
Add second part of mobile skeleton

### PR DESCRIPTION
**Pull request by AlejandraKeber**
Hello PR reviewer:
In this Milestone I did the following activities:

- Created a Feature branch
- Updated html and CSS files to finish the mobile version following the Figma template 1
- Used grid and flexbox to place elements on the page
- Checked and fixed lintern errors
Thanks for reviewing my PR!!!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-    <link rel="stylesheet" type="text/css" href="styles.css" />
-    <title>Portfolio setup and mobile first</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="styles.css" />
+  <title>Portfolio setup and mobile first</title>
 </head>
 <body>
     <div>
```
Comment (mgmediaweb) [inline index.html]: Kindly. I recommend you remove the code (DIV) that you do not use.

File: index.html [hunk 2]
```diff
@@ -36,5 +36,144 @@ <h6 class="footer">LET'S CONNECT</h6>
             </section>
         </main>
     </div>
+    <div>
```
Comment (mgmediaweb) [inline index.html]: Kindly. I recommend you remove the code (DIV) that you do not use.

File: index.html [hunk 3]
```diff
@@ -36,5 +36,144 @@ <h6 class="footer">LET'S CONNECT</h6>
...
+                    <h6 class="connect">LET'S CONNECT</h6>
+                    <ul class="socialicons2">
+                        <li><a class="image 1" href="https://www.twitter.com/home"><img src="./Images/Socialmedia/Vector.png" alt="twitter icon"></a></li>
+                        <li><a class="image 2" href="https://www.linkedin.com/in/alejandra-keber"><img src="./Images/Socialmedia/Linkedinicon.jpg" alt="likedin icon"></a></li>
+                        <li><a class="image 3" href="https://www.microverse.org/"><img src="./Images/Socialmedia/Vector(1).png" alt="microverse icon"></a></li>
+                        <li><a class="image 4" href="https://www.github.com/AlejandraKeber"><img src="./Images/Socialmedia/Vector(2).png" alt="github icon"></a></li>
+                        <li><a class="image 5" href="https://www.google.com/"><img src="./Images/Socialmedia/Vector(3).png" alt="vector3 icon"></a></li>
+                    </ul>
+                </div>
+                <div><button class="button-big">Get my resume</button></div>
```
Comment (mgmediaweb) [inline index.html:142]: Kindly. I recommend you remove the code (DIV) that you do not use.

File: index.html [hunk 4]
```diff
@@ -36,5 +36,144 @@ <h6 class="footer">LET'S CONNECT</h6>
...
+                </div>
+                <div>
+                    <h6 class="connect">LET'S CONNECT</h6>
+                    <ul class="socialicons2">
+                        <li><a class="image 1" href="https://www.twitter.com/home"><img src="./Images/Socialmedia/Vector.png" alt="twitter icon"></a></li>
+                        <li><a class="image 2" href="https://www.linkedin.com/in/alejandra-keber"><img src="./Images/Socialmedia/Linkedinicon.jpg" alt="likedin icon"></a></li>
+                        <li><a class="image 3" href="https://www.microverse.org/"><img src="./Images/Socialmedia/Vector(1).png" alt="microverse icon"></a></li>
+                        <li><a class="image 4" href="https://www.github.com/AlejandraKeber"><img src="./Images/Socialmedia/Vector(2).png" alt="github icon"></a></li>
+                        <li><a class="image 5" href="https://www.google.com/"><img src="./Images/Socialmedia/Vector(3).png" alt="vector3 icon"></a></li>
```
Comment (mgmediaweb) [inline index.html]: Kindly. class names should not have spaces (unless they are 2 different classes).

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @AlejandraKeber, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost2

## Highlights 💯 

- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

Kindly, I recommend you take care of the details so that it looks more like the original design and your website looks better

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @AlejandraKeber, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats


## Good Highlights ✅

- Implemented required changes
- Good commit messages
- PR is descriptive

## Optional suggestion

- I suggest that you include the _Acknowledgements_ and _License_ sections in your README file as provided in the recommended [README template](https://github.com/microverseinc/readme-template). This will make your documentation a lot more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**